components/delete_post_modal/index.js

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {bindActionCreators} from 'redux';
+import {connect} from 'react-redux';
+import {deletePost} from 'mattermost-redux/actions/posts';
+import {getCurrentTeam} from 'mattermost-redux/selectors/entities/teams';
+
+import DeletePostModal from './delete_post_modal.jsx';
+
+function mapStateToProps(state, ownProps) {
+    return {
+        ...ownProps,
```

Comment:
This is our fault because we do it everywhere (results of a giant copy/paste mistake :P ), but you don't actually need the `...ownProps` here at all. The props will get merged with what is returned here automatically